pkg/generators/openapi.go

```diff
@@ -844,15 +845,10 @@ func (g openAPITypeWriter) generateDescription(CommentLines []string) {
 		}
 	}
 
-	postDoc := strings.TrimLeft(buffer.String(), "\n")
-	postDoc = strings.TrimRight(postDoc, "\n")
-	postDoc = strings.Replace(postDoc, "\\\"", "\"", -1) // replace user's \" to "
-	postDoc = strings.Replace(postDoc, "\"", "\\\"", -1) // Escape "
-	postDoc = strings.Replace(postDoc, "\n", "\\n", -1)
-	postDoc = strings.Replace(postDoc, "\t", "\\t", -1)
-	postDoc = strings.Trim(postDoc, " ")
-	if postDoc != "" {
-		g.Do("Description: \"$.$\",\n", postDoc)
+	postDoc := strings.TrimSpace(buffer.String())
+	postDoc = strconv.Quote(postDoc)
```

Comment:
I think it'd be better to use `fmt.Sprintf("%#v", postDoc)` which formats a string as Go source

Reply:
Thanks for the suggestion! Changed to use Go-syntax representation in https://github.com/kubernetes/kube-openapi/pull/471/commits/5cb5474018333877c24c0fa1a73963d4c85dd075 :)